registries/docker-daemon/configure.sh

```diff
@@ -1,14 +1,21 @@
 #!/bin/bash
 
+set -o nounset
```

Comment:
rm

this script is sourced and should not modify the environment of the parent script